README.md

```diff
@@ -11,8 +11,9 @@ To learn how to use this template:
 
 If you just want to jump in and get started:
 
-1. Replace `function-template-go` with your function's name in `go.mod`,
-   `package/crossplane.yaml`, and any Go imports
+1. Replace `function-template-go` with your function in `go.mod`,
+   `package/crossplane.yaml`, and any Go imports. (You can also do this
+   automatically by running the `./init.sh <function-name>` script.)
```

Comment:
wouldn't the script do all the things below too? why just mention it here and not in the header?